src/background.ts

```diff
@@ -148,6 +146,15 @@ const store = new Store<ElectronStoreType>({
           defaultStyleId: defaultStyle.defaultStyleId,
         }))
       );
+
+      const outputSamplingRate: number =
+        // @ts-expect-error 削除されたパラメータ。
+        store.get("savingSetting").outputSamplingRate;
+      store.set(`engineSetting.${engineId}`, {
+        useGpu: store.get("useGpu"),
+        outputSamplingRate:
+          outputSamplingRate === 24000 ? "engineDefault" : outputSamplingRate,
+      });
```

Comment:
使わなくなるのでパラメータを削除しておくのはどうでしょう
 👀 
```suggestion
      const outputSamplingRate: number =
        // @ts-expect-error 削除されたパラメータ。
        store.get("savingSetting").outputSamplingRate;
      store.set(`engineSetting.${engineId}`, {
        useGpu: store.get("useGpu"),
        outputSamplingRate:
          outputSamplingRate === 24000 ? "engineDefault" : outputSamplingRate,
      });
      // @ts-expect-error 削除されたパラメータ。
      store.delete("savingSetting.outputSamplingRate");
      // @ts-expect-error 削除されたパラメータ。
      store.delete("useGpu");
```